File: code_challenge_solution/c#/143 Pascal's triangle.cs
Sure! Here's the C# code to generate Pascal's triangle:

```csharp
using System;

class Program
{
    static void Main()
    {
        int rows;

        Console.Write("Enter the number of rows for Pascal's triangle: ");
        rows = Convert.ToInt32(Console.ReadLine());

        PrintPascalTriangle(rows);
    }

    static void PrintPascalTriangle(int rows)
    {
        for (int i = 0; i < rows; i++)
        {
            int number = 1;
            Console.Write(new string(' ', rows - i));

            for (int j = 0; j <= i; j++)
            {
                Console.Write(number);
                Console.Write(' ');
                number = number * (i - j) / (j + 1);
            }

            Console.WriteLine();
        }
    }
}
```

Hope this helps!

File: code_challenge_solution/python/143 Pascal's triangle.py
def pascals_triangle(num_rows):
    triangle = []
    for row in range(num_rows):
        curr_row = []
        for col in range(row + 1):
            if col == 0 or col == row:
                curr_row.append(1)
            else:
                curr_row.append(triangle[row-1][col-1] + triangle[row-1][col])
        triangle.append(curr_row)
    return triangle

num_rows = int(input("Enter the number of rows: "))
triangle = pascals_triangle(num_rows)
for row in triangle:
    print(row)